src/estimate.coffee

```diff
@@ -105,6 +113,69 @@ updatePivotalTicket = ({ robot, res, projectId, ticketId, points, room }) ->
 
         if room then robot.messageRoom(room, msg) else res.send msg
 
+slackRequest = (robot, apiMethod) ->
+  url = "#{SLACK_BASE_URL}/#{apiMethod}"
+  robot.http(url)
+    .header("Content-Type", "application/json")
+
+blocks = () ->
```

Comment:
Just took an example from https://api.slack.com/tools/block-kit-builder which is amazing